### PR TITLE
NanoPlot tool wrapper added

### PR DESCRIPTION
**Pull request by egeogretmen**

### CODE REVIEW

File: tools/nanoplot/nanoplot.xml [hunk 1]
```diff
@@ -0,0 +1,498 @@
...
+            $customization.noN50
+            $customization.N50
+            #if $customization.title
+                --title '$customization.title'
+            #end if
+            $store
+            $raw
+            -o outputs &&
+            mv outputs/*.log outputs/NanoPlot_log.txt &&
+            if [[ -e outputs/NanoPlot-data.pickle ]]; then mv outputs/NanoPlot-data.pickle outputs/NanoPlot-data-pickle.binary; fi
```
Comment (bgruening) [inline tools/nanoplot/nanoplot.xml]: I don't think we need to support the pickle file case.

File: tools/nanoplot/nanoplot.xml [hunk 2]
```diff
@@ -0,0 +1,498 @@
...
+            $filter.barcoded
+            $customization.noN50
+            $customization.N50
+            #if $customization.title
+                --title '$customization.title'
+            #end if
+            $store
+            $raw
+            -o outputs &&
+            mv outputs/*.log outputs/NanoPlot_log.txt &&
```
Comment (bgruening) [inline tools/nanoplot/nanoplot.xml]: If we print the log file to stdout we do not need to put them as real output. I do not consider them as useful enough to store them as datasets.

So a simple `cat outputs/*.log` might be enough and will simplify our outputs of this tool dramatically.

File: tools/nanoplot/nanoplot.xml [hunk 3]
```diff
@@ -0,0 +1,498 @@
...
+                <option value="fasta">fasta</option>
+                <option value="fastq_rich">fastq_rich</option>
+                <option value="fastq_minimal">fastq_minimal</option>
+                <option value="summary">summary</option>
+                <option value="bam">bam</option>
+                <option value="cram">cram</option>
+                <option value="pickle">pickle</option>
+            </param>
+            <when value="fastq">
+                <param
```
Comment (bgruening) [inline tools/nanoplot/nanoplot.xml]: Uha, now I get your gitter question.

I think a simple `<param
                    type="data"
                    name="files"
                    format="fasta,fasta.gz,fasta.bz2,cram,bam"
                    multiple="true" />`

should be enough. No need to support all file formats, just be most standard once and keep your life simple :)

**Comment by egeogretmen** [comment]
@bgruening changed the test files, it should be ok now

**Comment by bgruening** [comment]
merge in IUC: https://github.com/galaxyproject/tools-iuc/pull/2100

**Pull request closed without merging**